docs/source/conf.py

```diff
@@ -19,7 +19,7 @@
 
 # -- Project information -----------------------------------------------------
 project = "sktime"
-copyright = "2019 - 2021 (BSD-3-Clause License)"
+copyright = "2019 - present (BSD-3-Clause License)"
```

Comment:
```suggestion
project_copyright = "2019 - present (BSD-3-Clause License)"
```

change constant to avoid builtin overriding? (ref. https://github.com/sphinx-doc/sphinx/issues/8132)